robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -472,6 +667,140 @@ public class ShadowAccessibilityNodeInfo {
   }
 
   @Implementation
+  public int getMovementGranularities() {
+    return movementGranularities;
+  }
+
+  @Implementation
+  public void setMovementGranularities(int movementGranularities) {
+    this.movementGranularities = movementGranularities;
+  }
+
+  @Implementation
+  public CharSequence getPackageName() {
+    return packageName;
+  }
+
+  @Implementation
+  public void setPackageName(CharSequence packageName) {
+    this.packageName = packageName;
+  }
+
+#if ($api >= 18)
+  @Implementation
+  public String getViewIdResourceName() {
+    return viewIdResourceName;
+  }
+
+  @Implementation
+  public void setViewIdResourceName(String viewIdResourceName) {
+    this.viewIdResourceName = viewIdResourceName;
+  }
+#end
+#if ($api >= 19)
+  @Implementation
+  public CollectionInfo getCollectionInfo() {
+    return collectionInfo;
+  }
+
+  @Implementation
+  public void setCollectionInfo(CollectionInfo collectionInfo) {
+    this.collectionInfo = collectionInfo;
+  }
+
+  @Implementation
+  public CollectionItemInfo getCollectionItemInfo() {
+    return collectionItemInfo;
+  }
+
+  @Implementation
+  public void setCollectionItemInfo(CollectionItemInfo collectionItemInfo) {
+    this.collectionItemInfo = collectionItemInfo;
+  }
+
+  @Implementation
+  public int getInputType() {
+    return inputType;
+  }
+
+  @Implementation
+  public void setInputType(int inputType) {
+    this.inputType = inputType;
+  }
+
+  @Implementation
+  public int getLiveRegion() {
+    return liveRegion;
+  }
+
+  @Implementation
+  public void setLiveRegion(int liveRegion) {
+    this.liveRegion = liveRegion;
+  }
+
+  @Implementation
+  public RangeInfo getRangeInfo() {
+    return rangeInfo;
+  }
+
+  @Implementation
+  public void setRangeInfo(RangeInfo rangeInfo) {
+    this.rangeInfo = rangeInfo;
+  }
+#end
+#if ($api >= 21)
+  @Implementation
+  public int getMaxTextLength() {
+    return maxTextLength;
+  }
+
+  @Implementation
+  public void setMaxTextLength(int maxTextLength) {
+    this.maxTextLength = maxTextLength;
+  }
+
+  @Implementation
+  public CharSequence getError() {
+    return error;
+  }
+
+  @Implementation
+  public void setError(CharSequence error) {
+    this.error = error;
+  }
+#end
+#if ($api >= 22)
+  @Implementation
+  public AccessibilityNodeInfo getTraversalAfter() {
```

Comment:
if (traversalAfter == null) {
      return null;
    }

    return obtain(traversalAfter);

Same for getTraversalBefore()